webapp/src/components/CollectionsTable.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <DynamicDataTable
+    :columns="collectionColumn"
+    :data="collections"
```

Comment:
This looks great, though I have a mild suggestion that we should retain the same component name as previously (so that the git commit history is retained on the file, essentially)

Reply:
Do you mean “PrimeDataTable”?

Reply:
No, sorry, I mean that `CollectionsTable.vue` should remain `CollectionTable.vue` -- I can make these tweaks